CHANGELOG

```diff
@@ -3,3 +3,4 @@
 [das-query-engine#223] Update log messages
 [#129] Create a new adapter called PostgresLobeDB
 [das-query-engine#214] Add retrieve_all_atoms method
+[#124] Refactor count_documents() in RedisMongoDB
```

Comment:
The messages here are not for our team but for outsiders. Ideally they should be a concise description of what changed after the PR have been merged. In this case something like "Changed count_atoms() to return more accurate numbers."